apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,25 @@
 # API Changelog
 
+## BFD-1383 Update V2 line item allowed charge amount mapping
```

Comment:
I'd replace this title to something like "Submitted charge amount updated from submitted to allowed charge amount"